anitya/lib/model.py

```diff
@@ -353,8 +353,8 @@ def __json__(self, detailed=False):
             version_url=self.version_url,
             version=self.latest_version,
             versions=self.versions,
-            created_on=time.mktime(self.created_on.timetuple()),
-            updated_on=time.mktime(self.updated_on.timetuple()),
+            created_on=time.mktime(self.created_on.timetuple()) if self.created_on else None,
```

Comment:
The ``created_on`` and ``updated_on`` fields get set when the object is being committed to the database, so if the object never got committed the fields won't be there. This occurred when the ProjectExists exception tries to turn the object to a dictionary.